docs/moodle.md

```diff
@@ -20,6 +20,53 @@ Go to `Plugins` tab, and click on `Manage tools` under `External Tools`:
   <td><img src="images/moodle_lti_select_admin_2.png" alt="Moodle LTI select admin view 2"/></td>
 </tr></table>
 
+##### Semi-automatic configuration
+
+A legacy LTI configuration tool is available, which presets the tool with a few parameters, but additional configuration is needed.
+
+To use it, in the `Tool URL…` field, paste the following URL: `https://marsha.education/lti/config.xml` and click on `Add Legacy LTI`
```

Comment:
```suggestion
To use it, in the `Tool URL` field, paste the following URL: `https://marsha.education/lti/config.xml` and click on `Add Legacy LTI`
```

Reply:
I kept `…` because it's displayed in the field
![image](https://user-images.githubusercontent.com/720491/169230756-f5c25453-5e95-4a82-82c5-60e233feed15.png)